cmd/fs-v1.go

```diff
@@ -72,15 +72,83 @@ func initMetaVolumeFS(fsPath, fsUUID string) error {
 
 }
 
+// Attempts to migrate old object metadata files to newer format
+//
+// i.e
+//    -------------------------------------------------------
+//    .minio.sys/buckets/<bucket_name>/<object_path>/fs.json
+//    -------------------------------------------------------
+//    .minio.sys/buckets/<bucket_name>/objects/<object_path>/fs.json
+//    -------------------------------------------------------
+//
+func migrateFSObjectMetadataDir(fsPath, fsUUID string) error {
```

Comment:
I can imagine that if a user has too many objects, he will press control-c and we will be in a middle of state. A better approach (unfortunately it also has its drawbacks) is to copy fs.json and not rename them. Once everything is finished, we upgrade fs format to v2.